src/app/clue.ts

```diff
@@ -202,6 +202,14 @@ export class Clue {
         // TODO: make-accusation button
         // if it's your turn
 
+           if (currPlayer.isTurn = true) {
```

Comment:
You're still assuming variables here and are using variable names from the backend code. `currPlayer` is not a variable/object in our frontend code. 

You should be using the `player` object when checking an attribute for this player (this player = player interacting with their browser).

Also, the condition is not the correct syntax. Should be using == instead of =. And you should also be checking if the returned element is already a Boolean. So something like:

`if ((this.player.isTurn) || (this.player.isTurn == true))`

Lastly, I didn't verify that the player has an attribute for .isTurn but you can check by seeing if the backend code's Player class's toJson() function includes a key for isTurn.